components/PolFooter/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import {
+  AboutUsBox,
+  ContactBox,
+  Container,
+  InformationBox,
+  InnerBox,
+  LinkItem,
+  LogoDiv,
+  MyAccountBox,
+  MyLink,
+  SocialLinksBox,
+  Title,
+  SocialLink,
+  SocialIcon
+} from "./PolFooter.styles";
+export type Link = {
+  url: string;
+  text: string;
+};
+export type SocialLink = {
+  icon: string;
+  url: string;
+};
+export type FooterData = {
+  contactInfo: {
+    company: string;
+    phone: string;
+    website: string;
+  };
+  information: Link[];
+  myaccount: Link[];
+  aboutus: Link[];
+  socials: SocialLink[];
+};
+export interface PolFooterProps {
+  data: FooterData;
+}
+const PolFooter: React.FC<PolFooterProps> = (props) => {
+  const { data } = props;
```

Comment:
const PolFooter... = ({ data }) => {...

Reply:
If we do it in this way, we can remove the line 40